src/main/java/org/eclipse/tractusx/ssi/lib/did/web/DidWebDocumentResolver.java

```diff
@@ -89,6 +89,7 @@ public DidDocument resolve(Did did) {
 
       return new DidDocument(json);
     } catch (IOException | InterruptedException e) {
+      Thread.currentThread().interrupt();
```

Comment:
Add error log

Reply:
If I try to add an error log, sonar complains with error **Either log this exception and handle it, or rethrow it with some contextual information** since we are rethrowing the exception, I didnot add log.